tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -327,6 +368,25 @@ <h3>No profile data was found.</h3>
     detached: function() {
       this.set('_isAttached', false);
     },
+    shouldDisableCaptureProfileButton: function(profilerServiceAddress, profileDuration) {
+      return !profilerServiceAddress || profileDuration < 0;
+    },
+    captureProfile: function() {
+      const captureProfileUrl = tf_backend.addParams(
+        tf_backend.getRouter().pluginRoute('profile', '/capture_profile'),
```

Comment:
stylistic nit: +2 whitespace for https://google.github.io/styleguide/jsguide.html#formatting-indent

Reply:
Done.

Reply:
Done